docs/src/components/HomepageFeatures.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import clsx from 'clsx';
+import styles from './HomepageFeatures.module.css';
+
+const FeatureList = [
+  {
+    title: 'OpenSource and free',
+    Svg: require('../../static/img/undraw_docusaurus_mountain.svg').default,
```

Comment:
wee need own pictures i think ...

Reply:
I replaced the images with some free icons.